gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -484,7 +484,8 @@ public synchronized void setActive(boolean active) {
           log.error("Exception encountered when shutting down DagManager threads.", e);
         }
       }
-    } catch (IOException e) {
+    } catch (RuntimeException | IOException e) {
```

Comment:
Catching the runtime exception here wouldn't do anything because we're rethrowing the runtime exception as a runtime exception

Reply:
Okay I will address this in a separate PR bc we need to actually change `GobblinServiceManager.handleLeadershipChange` to disconnect from Helix if the `DagManager.setActive` method throw an exception